code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -579,6 +579,14 @@ Please contact me on #coderbus IRC. ~Carnie x
 	var/standing	= image("icon"=female_clothing_icons["[t_color]_s"], "layer"=-layer)
 	return(standing)
 
+/mob/living/carbon/human/proc/get_overlays_copy(var/inhands=0)

```

Comment:
I have no idea how byond preproc treats undef here so this is possibly not working